packages/blade-mcp/knowledgebase/general/AvailableIcons.md

```diff
@@ -0,0 +1,416 @@
+- MagicCheckoutIcon
+- MagicKonnectIcon
+- MailIcon
+- MailOpenIcon
+- MapIcon
+- MapPinIcon
+- MaximizeIcon
+- MenuDotsIcon
+- MenuIcon
+- MessageCircleIcon
+- MessageSquareIcon
+- MicIcon
+- MicOffIcon
+- MinimizeIcon
+- MinusCircleIcon
+- MinusIcon
+- MinusSquareIcon
+- MobileAppIcon
+- MonitorIcon
+- MoonIcon
+- MoreHorizontalIcon
+- MoreIcon
+- MoreVerticalIcon
+- MoveIcon
+- MusicIcon
+- MyAccountIcon
+- NavigationIcon
+- NoSignalIcon
+- OctagonIcon
+- OffersIcon
+- OptimizerIcon
+- PackageIcon
+- PaperclipIcon
+- PauseCircleIcon
+- PauseIcon
+- PaymentButtonIcon
+- PaymentButtonsIcon
+- PaymentGatewayIcon
+- PaymentLinkIcon
+- PaymentLinksIcon
+- PaymentPagesIcon
+- PayoutLinkIcon
+- PayrollAddonsIcon
+- PayrollForCaIcon
+- PayrollForStartupOrSmeIcon
+- PercentIcon
+- PettyCashBudgetIcon
+- PhoneCallIcon
+- PhoneForwardedIcon
+- PhoneIcon
+- PhoneIncomingIcon
+- PhoneMissedIcon
+- PhoneOffIcon
+- PhoneOutgoingIcon
+- PictureInPictureIcon
+- PieChartIcon
+- PinIcon
+- PlayCircleIcon
+- PlayIcon
+- PlusCircleIcon
+- PlusIcon
+- PlusSquareIcon
+- PocketIcon
+- PosIcon
+- PowerIcon
+- PrinterIcon
+- PromptIcon
+- QRCodeIcon
+- RadioIcon
+- RayIcon
+- RazorpayIcon
+- RazorpayXIcon
+- RazorpayxPayrollIcon
+- RefreshIcon
+- RepeatIcon
+- ReportsIcon
+- ResizerIcon
+- RewindIcon
+- RotateClockWiseIcon
+- RotateCounterClockWiseIcon
+- RouteIcon
+- RoutesIcon
+- RupeeIcon
+- RupeesIcon
+- SaasIcon
+- SaveIcon
+- ScissorsIcon
+- SearchIcon
+- SendIcon
+- ServerIcon
+- SettingsIcon
+- SettlementsIcon
+- ShareIcon
+- ShieldIcon
+- ShoppingBagIcon
+- ShoppingCartIcon
+- ShuffleIcon
+- SidebarIcon
+- Signal1BarIcon
+- Signal2BarIcon
+- Signal3BarIcon
+- Signal4BarIcon
+- SignalIcon
+- SimCardIcon
+- SkipBackIcon
+- SkipForwardIcon
+- SlackIcon
+- SlashIcon
+- SlidersIcon
+- SmartCollectIcon
+- SmartphoneIcon
+- SolutionsIcon
+- SortIcon
+- SourceToPayIcon
+- SparklesIcon
+- SpeakerIcon
+- SquareIcon
+- StampIcon
+- StarIcon
+- StopCircleIcon
+- StorefrontIcon
+- SubscriptionsIcon
+- SunIcon
+- SunriseIcon
+- SunsetIcon
+- TabletIcon
+- TagIcon
+- TargetIcon
+- TaxPaymentsIcon
+- TestIcon
+- ThermometerIcon
+- ThumbsDownIcon
+- ThumbsUpIcon
+- TicketIcon
+- ToggleLeftIcon
+- ToggleRightIcon
+- TokenHqIcon
+- TopLeftRoundedCornerIcon
+- TopLeftSharpCornerIcon
+- TrademarkIcon
+- TrademarkRegisteredIcon
+- TransactionsIcon
+- TranslateIcon
+- TrashIcon
+- TrendingDownIcon
+- TrendingUpIcon
+- TriangleIcon
+- TvIcon
+- TwitterIcon
+- TypeIcon
+- UmbrellaIcon
+- UnderlineIcon
+- UnlockIcon
+- UpiAutopayIcon
+- UpiIcon
+- UploadCloudIcon
+- UploadIcon
+- UserCheckIcon
+- UserIcon
+- UserMinusIcon
+- UserPlusIcon
+- UserXIcon
+- UsersIcon
+- VendorPaymentsIcon
+- VideoIcon
+- VideoOffIcon
+- ViewLiveDemoIcon
+- VoicemailIcon
+- VolumeHighIcon
+- VolumeIcon
+- VolumeLowIcon
+- VolumeMuteIcon
+- WalletIcon
+- WatchIcon
+- WifiIcon
+- WifiOffIcon
+- WindIcon
+- WorldwideIcon
+- XCircleIcon
+- XSquareIcon
+- YoutubeIcon
+- ZapIcon
+- ZoomInIcon
+- ZoomOutIcon
```

Comment:
Not related to this, but we should modify  `generateIcons.mjs` to update this every time an icon is added.

Reply:
Oh right that's also a good idea. I was thinking of adding some build script before blade-mcp's deployment that reads the latest icons and adds it here. Will explore this later